net/libsrtp/Portfile

```diff
@@ -32,11 +32,23 @@ livecheck.url       https://github.com/cisco/libsrtp/releases
 livecheck.regex     {/v(\d+(?:\.\d+)*)}
 
 subport libsrtp1 {
-    version             1.5.4
-    revision            2
+    PortGroup           openssl 1.0
 
-    checksums           rmd160  9a02b2644839147dd794d368cede6cf7f23d75e4 \
-                        sha256  56a7b521c25134f48faff26b0b1e3d4378a14986a2d3d7bc6fefb48987304ff0
+    version             1.6.0
+    revision            0
+
+    # The subport is not compatible with 1.1 API, fails with error: field ‘ctx’ has incomplete type
+    # Related info: https://github.com/dun/munge/commit/d913c154425217f89b827e86686b777c9b3d0cf4
+    openssl.branch      1.0
+
+    checksums           rmd160  1dcbedcaa767d8fa2459270cc1902394260a0020 \
+                        sha256  1a3e7904354d55e45b3c5c024ec0eab1b8fa76fdbf4dd2ea2625dad2b3c6edde \
+                        size    1648862
+
+    platform powerpc {
+        configure.args-append \
+                        CPU_RISC=1
+    }
 
     livecheck.regex     {/v(1(?:\.\d+)*)}
 }
```

Comment:
the `distfiles` below shouldn't be needed either anymore when using the `github` PG